src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -467,24 +467,25 @@
         }
 
         .datagrid-column-handle {
```

Comment:
Can you confirm that the last column doesn't affect the width of the `.datagrid-table` element? I recall needing to rework the layout for this to keep it from impacting the table width.

Reply:
Do you remember the cases in which that specific behavior should be tested? I could cover that in `header-render.spec.ts` which checks all Datagrid headers together in an integrated manner. But as far as I see it visually on our demo app, it's behaving as expected. 

I have also a demo here: http://dnd-resize-header-demo.surge.sh/datagrids.

Reply:
There was an issue I saw a while ago where the column handle element was positioned outside the column and added some width to the overall header width when dragging. I looked at the demo and it looks ok.